ChatApp/Core/ChatView/ChatView.swift

```diff
@@ -7,24 +7,33 @@
 
 import UIKit
 
+protocol ChatViewDelegate: AnyObject {
+    func send(fromTop: Bool, messages: MessageEntity)
+    func reloadTableView()
+    func addActionsToTableView(at indexPaths: [IndexPath])
+    func scrollTableView(at indexPath: IndexPath)
+}
+
 class ChatView: UIView {
     
     //MARK: - Properties
     private lazy var tableView: UITableView = {
         let tableView = UITableView()
+        UITableView.appearance().separatorStyle = .none
+        UITableView.appearance().backgroundColor = .clear
```

Comment:
მარტო ამ tableViewზე არ გინდა ამ სტილის შეცვლა?
```suggestion
        tableView.separatorStyle = .none
        tableView.backgroundColor = .clear
```

Reply:
ეს პრობლემა ადრე განვიხილეთ, კი მხოლოდ მაგ თეიბლ ვიუს ვუცვლი მაგრამ მხოლოდ         tableView.separatorStyle = .none ვერ აღიქვამს(როგორც დავარესერჩე შესაძლოა ეფლის ბაგია) და სოლუშენი ეფიარენსი იყო